packages/cli/commands/sandbox/create.js

```diff
@@ -124,34 +117,19 @@ exports.handler = async options => {
   }
   const sandboxName = name || namePrompt.name;
 
-  let sandboxSyncPromptResult = true;
-  let contactRecordsSyncPromptResult = true;
+  let contactRecordsSyncPromptResult = false;
   if (!force) {
+    const isDevelopmentSandbox =
+      sandboxType === HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX;
     const syncI18nKey = 'lib.sandbox.sync';
-    const sandboxLangKey =
-      sandboxType === HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX
-        ? 'developer'
-        : 'standard';
-    const { sandboxSyncPrompt } = await promptUser([
-      {
-        name: 'sandboxSyncPrompt',
-        type: 'confirm',
-        message: i18n(`${syncI18nKey}.confirm.createFlow.${sandboxLangKey}`, {
-          parentAccountName: uiAccountDescription(accountId),
-          sandboxName,
-        }),
-      },
-    ]);
-    sandboxSyncPromptResult = sandboxSyncPrompt;
-    // We can prompt for contact records before fetching types since we're starting with a fresh sandbox in create
-    if (sandboxSyncPrompt) {
+
+    // Prompt to sync contact records for standard sandboxes only
+    if (!isDevelopmentSandbox) {
```

Comment:
Would it help to future proof this if we inverted this logic? I'm thinking about the scenario where we add new sandbox types way down the road (may not actually happen, but maybe still be worth guarding against?).